who_is_suspicious/R/statplot.R

```diff
@@ -0,0 +1,80 @@
+library(ggstatsplot)
+library(tidyverse)
+library(clock)
+
+GasTech_df <- read_csv("data/Final.csv")
+GasTech_df$Date <- date_time_parse(GasTech_df$Date,
+                                zone = "",
+                                format = "%m/%d/%Y")
+GasTech_df$CarID <- as_factor(GasTech_df$CarID)
+# 
+# GasTech_df = data.frame(GasTech)
```

Comment:
It is not necessary to add another line because tidyverse uses read_csv to read the file as a tibble which is what we need.